app/policies/outcome_coverage_policy.rb

```diff
@@ -0,0 +1,5 @@
+class OutcomeCoveragePolicy < ApplicationPolicy
+  def destroy?
+    true
```

Comment:
I need to change this. `user.manage_assessments?(record.outcome.course.department)`, perhaps?